.changeset/changelog.js

```diff
@@ -24,7 +24,7 @@ const defaultChangelogFunctions = {
     if (dependenciesUpdated.length === 0) return '';
 
     const changesetLinks = changesets.map(
-      (changeset) => `- Updated dependencies [${changeset.commit}]`,
+      (changeset) => `- **deps:** Updated dependencies [${changeset.commit}]`,
```

Comment:
Just curious, what necessitates this change? We have the old form copy-pasted in a few places 😓

https://github.com/search?q=org%3Aseek-oss+%22-+Updated+dependencies%22+language%3AJavaScript&type=code&l=JavaScript

Reply:
<img width="430" alt="image" src="https://github.com/seek-oss/skuba/assets/18017094/73dd31d6-c8a6-4ec5-b677-2b7ff266697b">

This gets outputted on the bottom of the skuba changelog

Reply:
https://github.com/seek-oss/skuba/actions/runs/8399947300/job/23006667152#step:11:35

Reply:
Hmm, interesting. I wonder if we should can this feature for now and return `''` if we're going to rely on manual changesets that provide more context.